docs/source/guide/calibrators.md

```diff
@@ -78,9 +78,20 @@ circuit = generate_rb_circuits(2, 10)[0]
 circuit.append(cirq.measure(circuit.all_qubits()))
 ```
 
-Instead of deciding what error mitigation technique and what options to use, we can ask Mitiq to determine the optimal error mitigation strategy based on the previous calibration. 
-We can obtain this by calling the `execute_with_mitigation` function and passing the `circuit`, along with the `Calibrator` object.
+Instead of deciding what error mitigation technique and what options to use, we can ask Mitiq to determine the optimal error mitigation strategy based on the previously performed calibration. 
+We can obtain this by calling the `execute_with_mitigation` function and passing the `circuit`, `Calibrator` object, and a new expectation value executor.
 
 ```{code-cell} ipython3
-execute_with_mitigation(circuit, cal, bitstring='00')
+def execute(circuit, noise_level=0.001):
```

Comment:
GitHub won't let me comment on lines already merged into master, but there should be a link to `executors.md` when they are first mentioned in this document.